### PR TITLE
fix: set CSS variable colours from vendor settings styles

### PR DESCRIPTION
**Pull request by danielgent**
One step closer!

image

Really not a million miles off scrapping login2 totally :sunglasses:

### CODE REVIEW

File: src/styles/input.css [hunk 1]
```diff
@@ -67,11 +67,10 @@
...
 }
 
-/* hardcoded kvartal styles */
 :root {
-  --primary-color: #4f3985;
-  --primary-hover: #ffffff;
-  --text-on-primary: #5f44a0;
+  --primary-color: #7d68f4;
+  --primary-hover: #7e69f4;
+  --text-on-primary: #ffffff;
```
Comment (danielgent) [inline src/styles/input.css:73]: I'm blindly copying what we have on login2!  *next* i need to actually set these with what we get from the vendor API :thinking:

File: src/utils/components/Layout.tsx [hunk 2]
```diff
@@ -7,6 +7,18 @@ type LayoutProps = {
...
 
+const globalDocStyle = (vendorSettings: VendorSettings) => {
+  const { style } = vendorSettings;
+  // cannot render CSS directly in JSX but we can return a template string
+  return `
+    body {
+      --primary-color: ${style.primaryColor};
+      --secondary-hover: ${style.primaryHoverColor};
+      --text-on-primary: ${style.buttonTextColor};
+    }
```
[the remark this answers is not in the document]
Comment (markusahlstrand) [reply src/utils/components/Layout.tsx:18]: There seems to be a helper for this:
https://hono.dev/helpers/css

**Review by markusahlstrand — APPROVED** [review]
LGTM!

File: src/utils/components/Layout.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielgent) [reply src/utils/components/Layout.tsx:18]: Nice *ALTHOUGH* we're still on v3 :smiling_face_with_tear: so we might not have this

Should we try and do the upgrade? I started out that PR on your hono-router package

**Pull request merged**